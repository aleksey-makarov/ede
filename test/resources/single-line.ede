{
  "var": true,
  "list": [1,2,3,4,5]
}
---
    {% if var %} this is a piece of text on a single line {% endif %}    

{% for n in list %}{{ n.value }} {% endfor %}
