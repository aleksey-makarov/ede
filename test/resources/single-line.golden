this is a piece of text on a single line

     this is a piece of text on a single line     

  1,2,3,4,5

